testsuites/syncgateway/functional/tests/test_channels.py

```diff
@@ -24,8 +24,11 @@ def test_channels_view_after_restart(params_from_base_test_setup, sg_conf_name):
     """
     - Add 10000 docs to Sync Gateway
     - Restart Sync Gateway (to flush channel cache)
+    - Add 1 doc
```

Comment:
To help people looking at this in the future, maybe change this to 
- Add 1 doc (to initialize the channel cache)

Reply:
Done